datavault-webapp/src/main/java/org/datavaultplatform/webapp/controllers/admin/AdminSchoolsController.java

```diff
@@ -51,14 +65,20 @@ public ModelAndView getSchoolRoleAssignmentsPage(@PathVariable("school") String
             throw new EntityNotFoundException(Group.class, schoolId);
         }
 
+        boolean canManageSchoolRoleAssignments = restService.getRoleAssignmentsForUser(principal.getName()).stream()
```

Comment:
Does this check for this particular school?

Reply:
No but it doesn't need to. There's 2 permissions associated with this method - `CAN_VIEW_SCHOOL_ROLE_ASSIGNMENTS` from the annotation at the top, and  `CAN_MANAGE_SCHOOL_ROLE_ASSIGNMENTS` from this statement. The one in the annotation is a school permission so the annotation checks it for the school. The one in this statement is an admin permission so if you have it, you can do stuff with any school